source/features/sticky-comment-header.css

```diff
@@ -0,0 +1,55 @@
+html:not([rgh-OFF-sticky-comment-header]) {
+	/* Issue body header */
+	[class^="IssueBodyHeader-module__IssueBodyHeaderContainer"],
+	/* Issue comment header */
+		[data-testid='comment-header'],
+	/* PR body/comment header */
+		.timeline-comment-header {
+		position: sticky;
+		top: calc(var(--base-sticky-header-height, 0px) + 56px);
+		z-index: 1;
+	}
+
+	/* Added by `show-names` */
+	.rgh-viewer-did-author,
+	.timeline-comment.current-user .timeline-comment-header,
+	[class*="IssueBodyHeader-module__viewerDidAuthor"] {
+		background: linear-gradient(var(--bgColor-accent-muted, fuchsia)), var(--rgh-background) !important;
```

Comment:
I don’t understand why this is using a gradient and `--rgh-background`

The color is defined by GitHub and it does not use gradients. Please keep things simple and native.
<img width="616" height="67" alt="Screenshot" src="https://github.com/user-attachments/assets/015597d7-72ef-4e4b-a1ac-d71ef9ae96c8" />